src/.goreleaser.yaml

```diff
@@ -18,13 +18,15 @@ builds:
           - -X 'github.com/alcionai/corso/src/internal/events.RudderStackDataPlaneURL={{.Env.RUDDERSTACK_CORSO_DATA_PLANE_URL}}'
 
 archives:
-  - name_template: "{{ .ProjectName }}_{{ .Tag }}_{{ .Os }}_{{ .Arch }}"
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  # this name template makes the OS and Arch compatible with the results of uname.
+  - name_template: >-
```

Comment:
is this leading line going to behave as expected? I'm not sure if it's GitHub workflow formatting magic or not. It doesn't seem like golang templating DSL

Reply:
The deprecation notice [seems to propose](https://goreleaser.com/deprecations/?h=replacement#archivesreplacements) this as the alternative to replacements.

Reply:
the overall text yes, but I'm wondering about the `>-` specifically which I don't think is part of golang's DSL for templates

Reply:
i see. 

I do see `>-` being used in several other packages. [Example](https://github.com/aquasecurity/trivy/blob/a9badeaba8bb9e908bce2f6f6defecf2f8515aeb/goreleaser.yml#L105)

Reply:
This came out of goreleasers autogenerated yaml file, so, I take that as a sign that it _must_ work.